src/renderers/testing/ReactTestRenderer.js

```diff
@@ -24,11 +24,13 @@ var ReactTestTextComponent = require('ReactTestTextComponent');
 var ReactTestEmptyComponent = require('ReactTestEmptyComponent');
 
 import type { ReactElement } from 'ReactElementType';
+import type { ReactInstance } from 'ReactInstanceType';
 
 type ReactTestRendererJSON = {
   type: string,
   props: { [propName: string]: string },
-  children: Array<string | ReactTestRendererJSON>,
+  children: null | Array<string | ReactTestRendererJSON>,
+  $$typeof?: any
```

Comment:
Flow kept giving me a warning about `$$typeof` if I didn't mark it as optional. I think it might be because we're using `Object.defineProperty`?